src/Node/index.tsx

```diff
@@ -29,11 +29,13 @@ type NodeProps = {
   onNodeMouseOver: NodeEventHandler;
   onNodeMouseOut: NodeEventHandler;
   subscriptions: object;
+  centerNode: (hierarchyPointNode: HierarchyPointNode<TreeNodeDatum>) => void | undefined;
```

Comment:
`void` is sufficient if the function does not return anything 👍 

```suggestion
  centerNode: (hierarchyPointNode: HierarchyPointNode<TreeNodeDatum>) => void;
```